dashboard/src/actions/charts.ts

```diff
@@ -156,3 +174,41 @@ export function getChartValues(
     }
   };
 }
+
+export function getChartUpdates(
+  name: string,
+  version: string,
+  appVersion: string,
+): ThunkAction<Promise<void>, IStoreState, null, ChartsAction> {
+  return async dispatch => {
+    try {
+      const chartsInfo = await Chart.listWithFilters(name, version, appVersion);
+      let update: IChartUpdate = {
+        checked: true,
```

Comment:
This is because of my suggestion https://github.com/kubeapps/kubeapps/pull/930#discussion_r249965136

Basically to differentiate between the case in which we have checked for updates and when we have not.

Some thoughts on the implementation though. 

I'd not make this action the one in charge of setting the checked flag.

What I'd do is to initialize that flag in the reducer ` updates: { checked: false },` and then set it to true when it receives `actions.charts.receiveChartUpdate` set it to true along with the updates.

I'd also potentially change the redux schema to  updates: { checked: boolean, items?: []IChartUpdates},` that will simplify the merge and also will allow you to not to add this empty update your are adding now.

Reply:
At the end I discarded the `checked` approach. It was blurring the code a lot. We can simply assume that if `update` is undefined it means that it hasn't been checked. I will add a comment to clarify it further.